src/openrct2/windows/footpath_scenery.c

```diff
@@ -300,7 +300,7 @@ static void window_footpath_scenery_scrollpaint(rct_window * w, rct_drawpixelinf
 	int y = 0, i = 0;
 	rct_string_id format = STR_BLACK_STRING;
 
-	gfx_fill_rect(dpi, dpi->x, dpi->y, dpi->x + dpi->width, dpi->y + dpi->height, ColourMapA[w->colours[1]].mid_light);
+	gfx_fill_rect(dpi, dpi->x, dpi->y, dpi->x + dpi->width, dpi->y + dpi->height, ColourMapA[w->colours[0]].mid_light);
```

Comment:
Not 100% sure, but this will probably fail if the window colour is set to translucent. `w->colours[0]` needs to be wrapped in `NOT_TRANSLUCENT()` macro.